tests/fixtures/iot/KL130(EU)_1.0_1.8.8.json

```diff
@@ -11,7 +11,7 @@
             "stopConnect": 0,
             "tcspInfo": "",
             "tcspStatus": 1,
-            "username": "#MASKED_NAME#"
+            "username": "user@example.com"
```

Comment:
Is this intentional?

Reply:
Yes, the `iot` protocol redactor masks it this way currently.